playbooks/roles/inventory/templates/hosts.j2

```diff
@@ -117,7 +136,7 @@ openshift_metrics_hawkular_hostname=metrics.{{_apps_dns}}
 openshift_metrics_cassandra_nodeselector={"node-role.kubernetes.io/infra": "true"}
 openshift_metrics_hawkular_nodeselector={"node-role.kubernetes.io/infra": "true"}
 openshift_metrics_heapster_nodeselector={"node-role.kubernetes.io/infra": "true"}
-
+b
```

Comment:
typo or is that lonely 'b' really supposed to be added?

Reply:
no, this accidently was added when navigating with the keybord mouse.
its gone now.